public/api/addPromotedContent.php

```diff
@@ -21,20 +22,113 @@
     // throws exception if current user is not an admin
     checkForAdmin($userId, $conn);
 
+    
```

Comment:
These queries are used solely for schema-migration. Since we don't have a schema-migration tool and this code is only run by admins, I guess it's OK to leave them in this PHP file for now. We should remove these schema-migration queries in a future commit after the schema is migrated.